test/poi_tests/test_poi_service.py

```diff
@@ -0,0 +1,67 @@
+# run with: python -m unittest test/poi_tests/test_poi_service.py
+import unittest
+from app.poi_api.poi_service import POI_Service
+from app.poi_api.poi_suggestion import POI_suggestions
+from app.poi_api.poi_errors import POINotFoundError, InvalidPOISuggestionError
+from firebase_admin import credentials, initialize_app, firestore
+
+
+class TestUser(unittest.TestCase):
+    @classmethod
+    def setUpClass(self):
+        cred = credentials.Certificate("app/key/serviceAccountKey.json")

```

Comment:
Is this a mock or a real firebase instance for testing? Cause that second one isn't gonna fly I think.

Reply:
oops